### PR TITLE
chore(report) : Add community bonding report

### PR DESCRIPTION
**Pull request by rajuljha**
## Project

Community Bonding report for @rajuljha for GSoC' 24

### Changes

Add community bonding report and add personal as well as project details.

### CODE REVIEW

File: docs/2024/ci-scanner/updates/2023-05-30.md
Comment (GMishx) [inline docs/2024/ci-scanner/updates/2023-05-30.md:1]: Please rename the file to report's date.

**Review by GMishx — CHANGES_REQUESTED** [review]

File: docs/2024/ci-scanner/updates/2023-05-30.md
Comment (rajuljha) [reply docs/2024/ci-scanner/updates/2023-05-30.md:1]: Done!

File: docs/2024/ci-scanner/index.md [hunk 1]
```diff
@@ -15,17 +15,33 @@ SPDX-FileCopyrightText: 2024 Rajul Jha <email.here>
...
+To be able to easily and continuously scan packages with
+fossology checks in CI pipelines, a docker image
+(fossology/fossology:scanner) capable of running license checks (using
+nomos or ojo) and keyword and copyright scans is available.
+This project aims to improve the CI Scanner Image in
+various aspects and numerous quality of life improvements, like
+highlighting the exact location of violation, ability to customize the
+keywords used by the scanner, and improving user experience – allow
+whitelisting from a custom location and ability to download and scan
+dependencies
```
Comment (avinal) [inline docs/2024/ci-scanner/index.md:36]: Please simplify this. Either write it in bullet points or separate the paragraphs. Try active voice.

File: docs/2024/ci-scanner/updates/2024-05-07.md
Comment (avinal) [inline docs/2024/ci-scanner/updates/2024-05-07.md:1]: Okay for community bonding, but for next updates, please try to include more from the discussion, like a few lines about the solution that was discussed/accepted.

**Review by avinal — CHANGES_REQUESTED** [review]
Slight change, rest looks good to me.

File: docs/2024/ci-scanner/updates/2024-05-07.md
Comment (rajuljha) [reply docs/2024/ci-scanner/updates/2024-05-07.md:1]: I have updated the project info as well as provided more meaningful community bonding report.

**Review by avinal — APPROVED** [review]
Looks good to me

**Comment by GMishx** [comment]
@rajuljha please update the commit message "Update report date" according to the contributing guidelines.

Hint: you can modify the message when you do interactive rebase.

**Comment by rajuljha** [comment]
> @rajuljha please update the commit message "Update report date" according to the contributing guidelines.
> 
> Hint: you can modify the message when you do interactive rebase.

Thanks for the tip! Please take a look if it is fine now?

**Review by GMishx — APPROVED** [review]

**Pull request merged**